### PR TITLE
Add container Host config, getting CPU / Memory reservations and limits in container info

### PR DESCRIPTION
**Pull request by gtardif**
Signed-off-by: Guillaume Tardif <guillaume.tardif@docker.com>

**What I did**
run `make protos` to update with latest protos from compose/cli

**Related issue**
syncing with https://github.com/docker/compose-cli/releases/tag/v0.1.22

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**

### CODE REVIEW

File: src/protos/containers/v1/containers.proto [hunk 1]
```diff
@@ -45,14 +45,26 @@ message Container {
 	string command = 4;
 	uint64 cpu_time = 5;
 	uint64 memory_usage = 6;
+	// kept for breaking change for now, moved to HostConfig
```
Comment (chris-crone) [inline src/protos/containers/v1/containers.proto:48]:
```suggestion
	// field deprecated, use equivalent in HostConfig
```

**Review by chris-crone — COMMENTED** [review]
Do we still fill the values for the deprecated fields in `Container`?

File: src/protos/containers/v1/containers.proto (hunk 1 above)
Comment (gtardif) [reply src/protos/containers/v1/containers.proto:48]: Yes, OK, will update the comment in the source in compose-cli, that will be included in a following PR (keeping sync with compose-cli releases is a bit of a constraint

Comment (gtardif) [reply src/protos/containers/v1/containers.proto:48]: And deprecated fields have a zero value.

**Pull request merged**